bgpd/bgp_ecommunity.c

```diff
@@ -1168,8 +1168,10 @@ char *ecommunity_ecom2str(struct ecommunity *ecom, int format, int filter)
 		bool unk_ecom = false;
 		memset(encbuf, 0x00, sizeof(encbuf));
 
+		if (number != -1 && (uint32_t)number != i)
```

Comment:
What does it mean (number)?

Reply:
I added some more information, and renamed it to index.